### PR TITLE
Add VinixMC.net

### PR DESCRIPTION
**Pull request by vinixmc**
## Type of change

- [ ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/vinixmc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "server_name": "vinixmc",
+  "nice_name": "VinixMC Network",
+  "direct_ip": "play.vinixmc.net",
+  "server_wildcards": [
+    "%.vinixmc.net"
+  ],
+  "social": {
+    "web_shop": "https://store.vinixmc.net",
+    "twitter": "16a2",
+    "discord": "https://discord.vinixmc.net"
```
Comment (maaaathis) [inline minecraft_servers/vinixmc/manifest.json:11]: Custom links are reserved for partners only; use a normal discord.gg/[INVITE] link instead

File: minecraft_servers/vinixmc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "server_name": "vinixmc",
+  "nice_name": "VinixMC Network",
+  "direct_ip": "play.vinixmc.net",
+  "server_wildcards": [
+    "%.vinixmc.net"
```
Comment (maaaathis) [inline minecraft_servers/vinixmc/manifest.json:6]: Doesn't look like a real wildcard
image

File: minecraft_servers/vinixmc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  ],
+  "social": {
+    "web_shop": "https://store.vinixmc.net",
+    "twitter": "16a2",
```
Comment (maaaathis) [inline minecraft_servers/vinixmc/manifest.json:10]: References not to a Minecraft server. Are you sure about that?

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by vinixmc** [comment]
Everything now is good, is a server wildcard required?

**Comment by maaaathis** [comment]
> Everything now is good, is a server wildcard required?

Nope, it's not required.

**Comment by vinixmc** [comment]
> > Everything now is good, is a server wildcard required?
> 
> Nope, it's not required.

How much does it take to verify?

**Comment by vinixmc** [comment]
> > Everything now is good, is a server wildcard required?
> 
> Nope, it's not required.


> > Everything now is good, is a server wildcard required?
> 
> Nope, it's not required.

What happened? when the server will be added?

**Comment by vinixmc** [comment]
> > > > Everything now is good, is a server wildcard required?
> > 
> > 
> > > 
> > 
> > 
> > > Nope, it's not required.
> > 
> > 
> > What happened? when the server will be added?
> 
> You need to remove the wildcard, if it's not a real wildcard.

I have removed the wildcard

**Comment by vinixmc** [comment]
> * JSON is invalid! Workflow is not able to check minecraft_servers/vinixmc/manifest.json
> 
> Please fix the issues by pushing **one** commit to the pull request to prevent too many automatic reviews.

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**